inference-engine/tests/functional/plugin/gna/shared_tests_instances/import_export_tests/import_reshape_permute_conv.cpp

```diff
@@ -14,13 +14,23 @@ namespace {
 class ImportReshapePermuteConvGNA : public ImportReshapePermuteConv {
 private:
     void exportImportNetwork() override {
-        executableNetwork.Export(fileName);
-        std::fstream inputStream(fileName, std::ios_base::in | std::ios_base::binary);
-        if (inputStream.fail()) {
-            FAIL() << "Cannot open file to import model: " << fileName;
+        static const char appHeader[] = "APPLICATION_HEADER";
```

Comment:
It is better to parameterize header presence

Reply:
Done. But in this case I had to also change base ImportNetworkTestBase class which affected also "Myriad" plugin tests. But now base implementation also has possibility to verify import/export with app headers